src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -125,12 +129,12 @@ class RuleNode extends React.Component<Props> {
   };
 
   getMailActionFields = (_: string, __: FormField) => {
-    const {data, organization, project} = this.props;
+    const {data, organization, project, disabled} = this.props;
     const isInitialized =
       data?.targetType !== undefined && `${data.targetType}`.length > 0;
     return (
       <MailActionFields
-        disabled={false}
+        disabled={disabled}
```

Comment:
It seems like the mailActionFields being disabled locks them in a permanent loading state:

![image](https://user-images.githubusercontent.com/9372512/90798064-7105a080-e2df-11ea-9c7d-0653aa3713c6.png)

It's not the biggest deal but could you take a look at any possible workarounds?

Reply:
solved thru https://github.com/getsentry/sentry/pull/20323